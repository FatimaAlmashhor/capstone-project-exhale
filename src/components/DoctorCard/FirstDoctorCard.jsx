import React from 'react';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import PropType from 'prop-types';
import '../../fontawesome';

export default function FirstDoctorCard({
  name,
  specialty,
  workplace,
  address,
  email,
  phone,
}) {
  return (
    <div className="container flex flex-col items-center px-5 py-16 mx-auto md:flex-row">
      <div className="w-5/6 lg:max-w-lg lg:w-full md:mb-0">
        <img
          className="object-cover object-center rounded"
          src="https://cdn.benchmark.pl/uploads/article/72855/MODERNICON/4f91fb01300b74ba42dcf080665285f3f56bd4ff.jpg"
          alt="Sunset in the mountains"
        />
      </div>
      <div className="flex flex-col items-center text-center lg:flex-grow lg:pl-24 md:pl-16 md:items-start md:text-left">
        <div className="px-6 py-4 text-left ">
          <div className="mb-2 text-2xl font-bold">
            <span>{name}</span>
          </div>
          <p className="text-xl text-gray-700">
            <span>Specialty :</span>
            {specialty}
          </p>
          <p className="text-xl text-gray-700">
            <span>Work Place :</span>
            {workplace}
          </p>
          <p className="text-xl text-gray-700">
            <span>Address :</span>
            {address}
          </p>
          <p className="text-xl text-gray-700">
            <span>Email :</span>
            {email}
          </p>
          <p className="text-xl text-gray-700">
            <span>Phone :</span>
            {phone}
          </p>
        </div>
        <div className="px-6 pt-4 pb-2">
          <FontAwesomeIcon
            className="text-lg text-black"
            icon={['fab', 'twitter']}
          />
          <FontAwesomeIcon className="mx-3 text-lg" icon={['fab', 'facebook']} />
          <FontAwesomeIcon className="mx-3 text-lg" icon={['fab', 'linkedin']} />
          <FontAwesomeIcon className="mx-3 text-lg" icon={['fab', 'instagram']} />
        </div>
      </div>
    </div>
  );
}

FirstDoctorCard.propTypes = {
  name: PropType.string.isRequired,
  specialty: PropType.string.isRequired,
  workplace: PropType.string,
  address: PropType.string,
  email: PropType.string,
  phone: PropType.number,
};
FirstDoctorCard.defaultProps = {
  workplace: 'dose not add work place yet',
  address: 'dose not add address yet',
  email: 'dose not add email yet',
  phone: 555555555555,
};
