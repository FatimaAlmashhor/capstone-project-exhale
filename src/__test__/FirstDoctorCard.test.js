import React from 'react';
import renderer from 'react-test-renderer';
import FirstDoctorCard from '../components/DoctorCard/FirstDoctorCard';

it('renders correctly', () => {
  const tree = renderer
    .create(<FirstDoctorCard name={' '} specialty= {' '} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});