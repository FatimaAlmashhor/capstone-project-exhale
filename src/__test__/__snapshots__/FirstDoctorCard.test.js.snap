// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container flex flex-col items-center px-5 py-16 mx-auto md:flex-row"
>
  <div
    className="w-5/6 lg:max-w-lg lg:w-full md:mb-0"
  >
    <img
      alt="Sunset in the mountains"
      className="object-cover object-center rounded"
      src="https://cdn.benchmark.pl/uploads/article/72855/MODERNICON/4f91fb01300b74ba42dcf080665285f3f56bd4ff.jpg"
    />
  </div>
  <div
    className="flex flex-col items-center text-center lg:flex-grow lg:pl-24 md:pl-16 md:items-start md:text-left"
  >
    <div
      className="px-6 py-4 text-left "
    >
      <div
        className="mb-2 text-2xl font-bold"
      >
        <span>
           
        </span>
      </div>
      <p
        className="text-xl text-gray-700"
      >
        <span>
          Specialty :
        </span>
         
      </p>
      <p
        className="text-xl text-gray-700"
      >
        <span>
          Work Place :
        </span>
        dose not add work place yet
      </p>
      <p
        className="text-xl text-gray-700"
      >
        <span>
          Address :
        </span>
        dose not add address yet
      </p>
      <p
        className="text-xl text-gray-700"
      >
        <span>
          Email :
        </span>
        dose not add email yet
      </p>
      <p
        className="text-xl text-gray-700"
      >
        <span>
          Phone :
        </span>
        555555555555
      </p>
    </div>
    <div
      className="px-6 pt-4 pb-2"
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-twitter fa-w-16 text-lg text-black"
        data-icon="twitter"
        data-prefix="fab"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-facebook fa-w-16 mx-3 text-lg"
        data-icon="facebook"
        data-prefix="fab"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-linkedin fa-w-14 mx-3 text-lg"
        data-icon="linkedin"
        data-prefix="fab"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-instagram fa-w-14 mx-3 text-lg"
        data-icon="instagram"
        data-prefix="fab"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </div>
  </div>
</div>
`;
